plugins/UsersManager/templates/index.twig

```diff
@@ -178,6 +178,7 @@
                 {% if showLastSeen is defined and showLastSeen %}
                 <th>{{ 'UsersManager_LastSeen'|translate }}</th>
                 {% endif %}
+                <th>token_auth</th>
```

Comment:
:white_check_mark: (marking for myself before updating the PR)

Using`.icon-reload` seems to fit nicely (displaying it already before token expansion).